__tests__/app.test.js

```diff
@@ -33,3 +33,38 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200 accepts an article_id which responds with the specified article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        const {article}  = body;
+          expect(article).toHaveProperty("article_id", expect.any(Number));
```

Comment:
As we're sending the request to a specific article id, should we be expecting any number here, or something else?